java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -985,7 +985,11 @@ public static OrderByArg desc(final int index, final boolean isNullSmallest) {
   }
 
   public static Aggregate count(int index) {
-    return Aggregate.count(index);
+    return Aggregate.count(index, false);
```

Comment:
I agree this is probably the choice that makes the most sense for `count(index)`.

Could you add function headers? This is not directly related to your change, but it would be nice to document what `count(index)` does in a header clearly.

Reply:
@nartal1 mind adding the function headers for the other aggregate ops while you are in there?

Reply:
Added function headers for other aggregate ops

Reply:
@nartal1 thanks for making changes, but could you add that the other functions are all skipping nulls? That was the part I was most interested in.

Reply:
Thanks @abellina . Updated it.